ui/src/app/app.component.ts

```diff
@@ -27,10 +27,13 @@ export class AppComponent {
     }
 }
 
+/**
+ * An HTML SELECT element with a list of choices.
+ */
 @Component({
     selector: 'app-select',
     template: `
-      <div *ngIf="ui_ctx[field].visible">
+      <div class="mb-3" *ngIf="ui_ctx[field].visible">
```

Comment:
This could be a default in the layout. We could add a top level `defaults:` in the yaml, but for now, that could be hardwired in the Python. We could use widget_class for this value.

Reply:
I was able to move the mb-3 class and related field.visible checkes to one place. I don't think the mb-3 would ever be changed, so I don't think this is a good case for making it configurable.